docs/howto/test.md

```diff
@@ -105,6 +105,30 @@ default schema path in an initializer.
 ActiveModelSerializers.config.schema_path = 'spec/support/schemas'
 ```
 
+### RSpec test helpers
+
+ActiveModelSerializers provides a `have_valid_schema` matcher to be used on your controller/request specs to
+validate the response against a [JSON Schema](http://json-schema.org/). Let's look at the equivalent setup for the example above.
+
+```ruby
+# rails_helper.rb
+require 'active_model_serializers/rspec'
+
+# spec/controller/posts_controller_spec.rb
+describe PostsController do
+  describe 'index' do
```

Comment:
should be `context 'GET /posts' do` IMO, since that's more descriptive and works well with autodoc tools

and then, per https://github.com/rails-api/active_model_serializers/pull/1947/files#r96015833 `get '/posts', {}, headers`